portfolio/src/main/webapp/style.css

```diff
@@ -122,5 +120,61 @@ h1{
     margin-left:10px;
     margin-right:10px;
     padding:4px;
+<<<<<<< HEAD
```

Comment:
Incomplete merge, make sure you resolve all conflicts when merging. That means going through any files that have been marked as conflicting, finding these tags, and choosing which version of the conflicting code you want to keep (or combining them correctly).